src/components/container/container.js

```diff
@@ -363,13 +363,17 @@ export default class Container extends UIObject {
   }
 
   disableMediaControl() {
-    this.mediaControlDisabled = true
-    this.trigger(Events.CONTAINER_MEDIACONTROL_DISABLE)
+    if (this.mediaControlDisabled !== true) {
```

Comment:
I agree with @leandromoreira. Using `mediaControlEnabled` seems easier to read, and his suggestion gets rid of the weird `!== true` comparison for a boolean value, since we would never have an `undefined` value.